src/org/omegat/gui/common/EntryInfoPane.java

```diff
@@ -97,6 +92,7 @@ public void clear() {
         UIThreadsUtil.mustBeSwingThread();
         setText(null);
         scrollRectToVisible(new Rectangle());
+        setComponentOrientation(BiDiUtils.getInitialOrientation());
```

Comment:
This is the main change for that PR, it enables this component to handle RTL languages.